src/components/Colors.tsx

```diff
@@ -15,9 +15,11 @@ interface Props {
 function Colors(props: Props) {
   const { colors } = props;
   const [curColor, setCurColor] = useState('');
+  const [message, setMessage] = useState('');
```

Comment:
whenever someone uses setMessage, it makes me sad because I made a really nice messaging system :(

Reply:
Check this out: https://github.com/CS490-section03-group0/project3/pull/62/files#diff-18a05e32cfcc0b24161ab4950b3460c845f7861624fa17ce3e4107be071e8699